pkg/apis/k0s.k0sproject.io/v1beta1/dualstack_test.go

```diff
@@ -1,36 +0,0 @@
-package v1beta1
-
-import (
-	"github.com/stretchr/testify/require"
-	"testing"
-)
-
-func TestAddDualStackArguments(t *testing.T) {
```

Comment:
for the context: deleted because this test is now a part of the feature gate helper test